install.sh

```diff
@@ -49,35 +60,47 @@ fi
 echo "-------------------------"
 echo "${color_green}[#]${color_normal} ${color_orange}makedeb Installer${color_normal} ${color_green}[#]${color_normal}"
 echo "-------------------------"
-
 echo
+
+if [[ "${DEBIAN_FRONTEND}" == "noninteractive" || "${NONINTERACTIVE}" == "1" || "${CI}" == "1" ]]; then
+    msg "Enabling noninteractive mode"
+    noninteractive_mode=1
+    export DEBIAN_FRONTEND=noninteractive
+    additional_apt_args=-y
```

Comment:
Can you make this an array so we can pass these as a list of arguments to the `apt` commands?